### PR TITLE
4492 fix some warnings

### LINKED ISSUE #4492: fix some compile warnings
## What went wrong? 😲

Develop won't compile due to missing struct fields in test data 

## Your environment 🌱

- Open mSupply Version: `develop`

### PR DESCRIPTION
**Pull request by Chris-Petty**
Fixes #4492

# 👩🏻‍💻 What does this PR do?

Another PR fixed the same stuff. Just some compile time warnings fixed now.

### CODE REVIEW

**Comment by Chris-Petty** [comment]
ded

**Review by roxy-dao — CHANGES_REQUESTED** [review]
Develop compiles fine for me...

also tests are still failing (: 

Actions has been failing for backend for a while now... Doesn't stop merge if it's been approved 🤷🏻

**Comment by Chris-Petty** [comment]
What's the point of tests if they're failing and we ignore it 😕

**Comment by roxy-dao** [comment]
> What's the point of tests if they're failing and we ignore it 😕

https://github.com/msupply-foundation/open-msupply/issues/2900

**Comment by Chris-Petty** [comment]
Omg I swear my PR passes tests it's just doing `#2900` now @roxy-dao pls approve 😆

**Review by roxy-dao — APPROVED** [review]
Thanks Chris!

File: server/service/src/sync/test/test_data/rnr_form_line.rs [hunk 1]
```diff
@@ -50,7 +50,6 @@ fn rnr_form_line_1() -> RnRFormLineRow {
         requested_quantity: 0.0,
         comment: None,
         confirmed: false,
-        previous_average_monthly_consumption: 0.0,
```
Comment (Chris-Petty) [inline server/service/src/sync/test/test_data/rnr_form_line.rs:52]: Weird I swear I did remove this 🤔 Maybe played too dirty with my commit munging

Comment (roxy-dao) [reply server/service/src/sync/test/test_data/rnr_form_line.rs:52]: You did... it got duplicated XD

**Pull request merged**